desc/objectives/_free_boundary.py

```diff
@@ -104,15 +104,17 @@ def __init__(
         normalize_target=True,
         loss_function=None,
         deriv_mode="auto",
-        grid=None,
+        eval_grid=None,
         field_grid=None,
         field_fixed=False,
         name="Vacuum boundary error",
         jac_chunk_size=None,
+        **kwargs,
     ):
+        self._eval_grid = parse_argname_change(eval_grid, kwargs, "grid", "eval_grid")
```

Comment:
Currently, if the user passed in `grid=foo`, then this will correctly assign grid to `self._eval_grid` however, the line below
`self._eval_grid = eval_grid` 
would then overwrite it with `None`

I think the fix is just make this `eval_grid = parse_argname_change(...` and then the line below will work as intended.